src/math/Triangle.js

```diff
@@ -103,7 +103,12 @@ class Triangle {
 
 	static getInterpolation( point, p1, p2, p3, v1, v2, v3, target ) {
 
-		this.getBarycoord( point, p1, p2, p3, _v3 );
+		if ( this.getBarycoord( point, p1, p2, p3, _v3 ) === null ) {
+
+			target.set( 0, 0, 0 );
+			return null;
```

Comment:
I think we also have to update the docs of `getUV()` now since with this PR, `getUV()` can also return `null` now.